packages/node/src/sdk/index.ts

```diff
@@ -221,6 +222,15 @@ function getClientOptions(
         ? true
         : options.autoSessionTracking;
 
+  if (options.spotlight == null) {
+    const spotlightEnv = envToBool(process.env.SENTRY_SPOTLIGHT, { strict: true });
+    if (spotlightEnv == null) {
+      options.spotlight = process.env.SENTRY_SPOTLIGHT;
+    } else {
+      options.spotlight = spotlightEnv;
+    }
```

Comment:
can we simplify this if stmt? I think we if `spotlightEnv` is null, we don't need to assign `options.spotlight at all, right? 

```suggestion
    if (spotlightEnv != null) {
      options.spotlight = spotlightEnv;
    }
```

Reply:
That's not really the case here. Looks like I need better variable naming. `spotlightEnv` would be `null` if `$SENTRY_SPOTLIGHT` is neither falsy nor truthy such as a custom URL. In that case we should just pass whatever the value in `$SENTRY_SPOTLIGHT` is.

I can put that logic into the helper function too but I feared it might get even more confusing.

Reply:
Oh sorry, I totally forgot that `options.spotlight` can also be a string for the sidecar url. You're right of course! 
No need for a helper function from my PoV -  we can leave it as is.